### DISCUSSION TITLE
Using blog plugin as home page

### DISCUSSION THREAD

**Discussion by pitosalas** [squidfunk/mkdocs-material#6590] [Q&A] [answered]
I would like my blog index page to be the index page of my site. How can I do that? I thought 

plugins:
    - blog
        blog_dir: "."

Would work but that gives an exception. Am i on the right track?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @pitosalas,
what exception is given? The `blog_dir` option needs to be properly indented, also as you're changing the base path from `blog` to `.` you need to also move all of the `docs/blog` files into `docs/` one level higher. 
You can refer to this answer with an example if needed https://github.com/squidfunk/mkdocs-material/discussions/6430#discussioncomment-7753446

**Comment by pitosalas** [comment]
Thanks, that was the clue I needed. I had a problem with the indentation. The issue can be closed. Pito Salas Computer ScienceBrandeis Univeristy On Jan 4, 2024, at 2:31 AM, Kamil Krzyśków ***@***.***> wrote:﻿
Hello @pitosalas,
what exception is given? The blog_dir option needs to be properly indented, also as you're changing the base path from blog to . you need to also move all of the docs/blog files into docs/ one level higher.
You can refer to this answer with an example if needed #6430 (comment)
